packages/sui-test-domain/src/describers/index.js

```diff
@@ -0,0 +1,3 @@
+import {describeOnServer, describeOnClient} from './serverDescribers'
+
+export {describeOnServer, describeOnClient}
```

Comment:
Remember to change this to auto detect node instead of rely on the package.json config